### DISCUSSION TITLE
Conditional Use of Wiki Updater only if Generator was successful

### DISCUSSION THREAD

**Discussion by JonathanCorbett** [rfennell/AzurePipelines#1950] [Q&A]
I have a shared task group which handles the generation of release notes for the current environment and the next environment if Production (using the overrideStageName flag). The same task group then pushes to 1 of 2 wiki pages and all looks good in general.

Due to the fact that I have the stopOnRedploy flag selected on the first task, this completes (correctly) with a warning and no release notes file. Unfortunately the next step then fails as there is no release notes md file.

Is it possible to reference the stopOnRedeploy variables from the Generator task as an Output Variable or by other means into the second task as a conditional run statement, so the second task doesn't run and fail? I've tried a few variations of accessing the variable in the second step but no luck so far, and this must be a common request so I'm sure a decent solution has been found.

Mainly looking for the syntax of the variable and how to reference/use it in the next step.

**Comment by rfennell** [comment]
Unfortunately, the `stopOnRedeploy` is not an output variable so can't be consumed by subsequent tasks.

However, could you not create a pipeline variable and use this to set the `stopOnRedeploy` and also use it in the subsequent tasks condition statement?
